src/components/Footer.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Divider from 'material-ui/Divider';
+
+const Footer = () => (
+  <footer
+    className="restrict-width"
+    style={{
+      height: '10%',
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+      justifyContent: 'center',
+    }}
+  >
+    <Divider style={{ width: '100%', marginTop: '10px' }} />
+    <p style={{ display: 'flex', flexGrow: 1, alignItems: 'center' }}>Code Chrysalis © 2017</p>
+  </footer>
```

Comment:
@bill-feng Is there a reason to put inline styles here instead of writing in css?

Reply:
I did this for the Footer because I don't think we will be changing the footer much, and it's very little code so this saves the trouble of having another file.